routes/comment.js

```diff
@@ -13,8 +14,8 @@ const {
 const { create, list, archive } = CommentController;
 const router = express.Router();
 
-router.get('/:id/comments', ensureAuthenticated, checkArticleId, articleExist, list);
-router.post('/:id/comments', ensureAuthenticated, checkCommentContent, articleExist, create);
-router.delete('/:articleId/comments/:commentId', ensureAuthenticated, checkCommentId, checkCommentParams, archive);
+router.get('/:identifier/comments', ensureAuthenticated, checkArticle, checkArticleId, list);
+router.post('/:identifier/comments', ensureAuthenticated, checkArticleId, checkCommentContent, checkArticle, create);
+router.delete('/:identifier/comments/:commentId', ensureAuthenticated, checkArticle, checkCommentId, checkCommentParams, archive);
```

Comment:
I think it is better you left `identifier` as `articleId`. Peter had already resolved what you have caused an issue.

Reply:
for now, I will leave it as Identifier because the articleMiddleware receives identifier based on the refactor of Victor's code with the TTL.